themis/regulations/apple-crypto-regulations.md

```diff
@@ -1,9 +1,9 @@
 ---
 weight: 1
-title:  App Store requirements
+title: App Store requirements
```

Comment:
Since this page is going to be about US cryptography export regulations, I think it should have an appropriate title.

```suggestion
title: US crypto export regulations
```